msal-objc-articles/cloud-instance-discovery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Cloud instance discovery: Instance aware flow"
+description: "Some apps might need to work with multiple cloud instances at the same time depending on the account. In that case, instance aware flow might be useful."
+services: active-directory
+author: Dickson-Mwendia
+manager: CelesteDG
+
+ms.service: active-directory
+ms.subservice: develop
+ms.topic: conceptual
+ms.workload: identity
+ms.date: 09/22/2023
+ms.author: dmwendia
+ms.reviewer: oldalton, brianmel
+ms.custom: aaddev
+---
+
+# Cloud instance discovery: Instance aware flow
```

Comment:
Do we want to publish this article? Looks like it's more of an internal feature used in the air-gapped clouds.

@oldalton @iambmelt @localden

Reply:
We do not need to document this publicly, this is 1P-only capability. We have docs for the 1P scenario here:

https://review.learn.microsoft.com/en-us/identity/microsoft-identity-platform/sovereign-clouds?branch=main#azure-ad-instance-discovery

Reply:
Great, thanks for the clarification here.